lib/appsignal.rb

```diff
@@ -195,7 +195,7 @@ def in_memory_log
     end
 
     def logger
-      @logger ||= Logger.new(in_memory_log).tap do |l|
+      @logger ||= Appsignal::Logger.new(in_memory_log).tap do |l|
```

Comment:
This is only the in memory logger (which is used on start before AppSignal is active). We also need to use the `Appsignal::Logger` in the `start_stdout_logger` and `start_file_logger` methods.

Probably a good idea to add an assertion in the `appsignal_spec` somewhere if we're using the `Appsignal::Logger` class for logging.

Reply:
Fixed and added